libs/blocks/tabs/tabs.js

```diff
@@ -84,13 +87,39 @@ function initTabs(elm, config, rootElem) {
   if (config) configTabs(config, rootElem);
 }
 
-let initCount = 0;
+function handleDeferredImages(e) {
+  const images = e.querySelectorAll('img[loading="lazy"]');
+  images.forEach((img) => {
+    /* c8 ignore next */
+    img.removeAttribute('loading');
+  });
+  e.removeEventListener('milo:deferred', handleDeferredImages, true);
+}
 
 const init = (block) => {
   const rootElem = block.closest('.fragment') || document;
   const rows = block.querySelectorAll(':scope > div');
+  const parentSection = block.closest('.section');
   /* c8 ignore next */
   if (!rows.length) return;
+  block.addEventListener('milo:deferred', handleDeferredImages(block), true);
+
+  // Tab Config
+  const config = {};
+  const configRows = [].slice.call(rows);
```

Comment:
I'm not familiar with this pattern. Is it the same as `[...rows]`?

Reply:
Both methods are the same and can be used based on personal preference. I might update this to use the spread operator to keep more consistent w/ other block methods. consistency is key.